mlflow/java/client/src/main/java/org/mlflow/tracking/RunsPage.java

```diff
@@ -0,0 +1,91 @@
+package org.mlflow.tracking;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import org.mlflow.api.proto.Service.*;
+
+public class RunsPage implements Page<Run> {
+
+    private final String token;
```

Comment:
@aarondav, do you know why we have 2 space indents for the Java MLflow code? If we want to do this, maybe this should be 2 space indented too.